config.go

```diff
@@ -281,6 +287,8 @@ type Config struct {
 	MinBackoff        time.Duration `long:"minbackoff" description:"Shortest backoff when reconnecting to persistent peers. Valid time units are {s, m, h}."`
 	MaxBackoff        time.Duration `long:"maxbackoff" description:"Longest backoff when reconnecting to persistent peers. Valid time units are {s, m, h}."`
 	ConnectionTimeout time.Duration `long:"connectiontimeout" description:"The timeout value for network connections. Valid time units are {ms, s, m, h}."`
+	SOCKS             string        `long:"socks" description:"The host:port that SOCKS5 proxy to use for any outbound connections not going over Tor"`
+	NoProxyTargets    string        `long:"no-proxy-targets" description:"Comma-separated values specifying hosts that should bypass the clearnet proxy. Each value is either an IP address, a CIDR range, a zone (*.example.com) or a host name (localhost). A best effort is made to parse the string and errors are ignored. (default: localhost,127.0.0.0/8,::1/128)"`
```

Comment:
For flags with multiple possible values we usually use a slice internally and then use the singular form in the flag name and mention it can be used multiple times.
So for example:
```
NoProxyTargets []string `long:"no-proxy-target" description:"Specify a host/range that should bypass the clearnet proxy. ... Can be specified multiple times"`
```

Then use like this:
```
--no-proxy-target=localhost
--no-proxy-target=127.0.0.0/8
--no-proxy-target=::1/128
```